source/_lovelace/picture-glance.markdown

```diff
@@ -25,30 +23,80 @@ You can also use `camera` domain entities to use that as image. You can also use
 Screenshot of the picture glance card.
 </p>
 
-**Options**
+{% linkable_title Options %}
 
-| Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
-| type | string | **Required** | `picture-glance`
-| image | string | **Required** | URL of an image
-| entities | list | **Required** | Entity id's
-| navigation_path | string | Optional | Path of URL to use in navigation
-| camera_image | string | Optional | camera domain entity_id 'camera.demo_camera'
-| state_image | object | Optional| See `state_image` object structure.
-| entity | list | Optional | An entity to use for state_image state
-| title | string | Optional | Card title
+{% configuration %}
+type:
+  required: true
+  description: picture-glance
+  type: string
+image:
+  required: true
+  description: URL of an image
+  type: string
+entities:
+  required: false
+  description: Entity id's
+  type: list
+  default: None
+navigation_path:
+  required: false
+  description: Path of URL to use in navigation
```

Comment:
should be something like "navigate to HA frontend url on click"

Reply:
I dont agree as the URL does not have to be to the HA frontend url, it could be another url.